test/e2e/build_test.go

```diff
@@ -41,6 +41,35 @@ var _ = Describe("Podman build", func() {
 		Expect(session).Should(ExitCleanly())
 	})
 
+	It("podman image prune should clean build cache", func() {
+		// try writing something to persistent cache
+		session := podmanTest.Podman([]string{"build", "-f", "build/buildkit-mount/Containerfilecachewrite"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		// try reading something from persistent cache
+		session = podmanTest.Podman([]string{"build", "-f", "build/buildkit-mount/Containerfilecacheread"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+		Expect(session.OutputToString()).To(ContainSubstring("hello"))
+
+		// Prune build cache
+		session = podmanTest.Podman([]string{"image", "prune", "-f", "--build-cache"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
```

Comment:
Sorry I just realized the cache is global not actually in the container storage, this means the it cannot be run in parallel without causing flakes for other tests doing `--mount=type=cache`. Now I do see that there is only one today in `podman system reset must clean host shared cache` which is already declared serial so it doesn't cause any problems as of today but as soon as somebody adds a second test it can start flaking.

So please add the "Serial" decorator for this test as well as this might become a problem at some point otherwise.

Reply:
Done.